nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -213,9 +220,99 @@ public static String serializeSymbol(@Nullable Symbol symbol) {
+      if (name.isEmpty()) {
+        name = methodSymbol.owner.enclClass().getSuperclass().asElement().getSimpleName();
+      }
+      sb.append(name);
+    } else {
+      // For methods, we use the name of the method.
+      sb.append(methodSymbol.getSimpleName());
+    }
+    // Index for iteration in visiting method parameters. Used to determine if visiting the last
+    // parameter.
+    AtomicInteger index = new AtomicInteger(1);
+    sb.append(
+        methodSymbol.getParameters().stream()
+            .map(
+                parameter -> {
+                  // varargs is only defined on the method and not on the parameter. This
+                  // information is lost while visiting the last parameter in the method if it is
+                  // varargs and the parameter is visited as a normal array type. We need to
+                  // manually convert the last parameter to varargs if the method is varargs.
+                  Type type =
+                      // if the parameter is the last parameter,
+                      (index.getAndIncrement() == methodSymbol.getParameters().size()
+                              // if the method is varargs,
+                              && methodSymbol.isVarArgs()
+                              // for safety of casting, check if the parameter is an array type
+                              && parameter.type instanceof Type.ArrayType)
+                          ? ((Type.ArrayType) parameter.type).makeVarargs()
+                          : parameter.type;
+                  return type.accept(
+                      new Types.DefaultTypeVisitor<String, Void>() {
+                        @Override
+                        public String visitWildcardType(Type.WildcardType t, Void unused) {
+                          StringBuilder sb = new StringBuilder();
+                          sb.append(t.kind);
+                          if (t.kind != BoundKind.UNBOUND) {
+                            sb.append(t.type.accept(this, null));
+                          }
+                          return sb.toString();
+                        }
+
+                        @Override
+                        public String visitCapturedType(Type.CapturedType t, Void s) {
+                          return t.wildcard.accept(this, null);
+                        }
+
+                        @Override
+                        public String visitClassType(Type.ClassType t, Void unused) {
+                          StringBuilder sb = new StringBuilder();
+                          sb.append(t.tsym);
+                          if (t.getTypeArguments().nonEmpty()) {
+                            sb.append('<');
+                            sb.append(
+                                t.getTypeArguments().stream()
+                                    .map(arg -> arg.accept(this, null))
+                                    .collect(joining(",")));
+                            sb.append(">");
+                          }
```

Comment:
I don't think type arguments are necessary to uniquely identify a method.  Do we need them in the string for some reason?

Reply:
Yes, it is not required. But changing that will require updating 2 unit test outputs and a change in Injector. Please let me know if I should remove type argument serialization. I am fine with changing Injector, but just wanted to confirm if we should change unit test output in this PR as well. Thank you.

Reply:
@msridhar At this moment, I think this will simplify both code in NullAway and Injector. So If you agree changing failing unit tests outputs in this PR, I will change it.

Reply:
My personal preference is to have less code to maintain.  So, if the code change to the injector is not too significant / difficult, I'd prefer not to serialize the type arguments.

On the other hand, if you feel that serializing type arguments is important for readability, or some other reason that I am missing, I don't strongly object to keeping them

Reply:
Yes, I agree and I prefer to remove it as well. It will reduce complexity and worths the update on Injector as well. 424bdb5cc98a87d335d76571df5410762f08ce0a